components/gc-featured-link/reports/pre-a11y-2.json

```diff
@@ -0,0 +1,589 @@
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:labels-or-instructions",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:error-suggestion",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:error-prevention-legal-financial-data",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:parsing",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:passed",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:name-role-value",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
+        },
+        {
+            "earl:test": "WCAG21:status-messages",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "",
+            "dct:modified": null,
+            "acr:severity": "acr:noSeverity",
+            "acr:relevancy": "wbv:acr#noRelevancy",
+            "earl:mode": "earl:manual",
+            "@type": ["earl:TestResult", "earl:Assertion"]
```

Comment:
Suggestion: Considering that you already did a pre-asessment which it do identify the SC that is not applicable, you don't need to repeat it here.

```suggestion
```